### ISSUE TITLE
add tensorflow_gpu-2.0 for conda 10.1

### ISSUE THREAD

**Issue by beoy** [fo40225/tensorflow-windows-wheel#99]
hello, today google release  tensorflow 2.0, but my pc have'not install it because i using conda 10.1, please build a 10.1 version for users.thank you

**Comment by henrysky** [comment]
> hello, today google release tensorflow 2.0, but my pc have'not install it because i using conda 10.1, please build a 10.1 version for users.thank you

btw there is no more tensorflow-gpu, tensorflow will automatically detect whether to use GPU or not.

**Comment by liyike1988** [comment]
Can tensorflow 2.0 support CUDA 3.0?

**Comment by Cruel-Summer** [comment]
> btw there is no more tensorflow-gpu, tensorflow will automatically detect whether to use GPU or not.

I don't think you are right. Here is what on [https://tensorflow.google.cn/install/pip](https://tensorflow.google.cn/install/pip):
### Install TensorFlow with pip
#### TensorFlow 2 packages are available
- tensorflow —Latest stable release for CPU-only
- tensorflow-gpu —Latest stable release with GPU support (Ubuntu and Windows)

**Comment by fo40225** [comment]
> > hello, today google release tensorflow 2.0, but my pc have not install it because i using cuda 10.1, please build a 10.1 version for users.thank you
> 
> btw there is no more tensorflow-gpu, tensorflow will automatically detect whether to use GPU or not.

I'm trying to build tensorflow 2.0 with cuda 10.1.

Only `tensorflow==1.15.0` pip package have GPU support, 2.0 does not.

https://groups.google.com/a/tensorflow.org/forum/#!topic/developers/iRCt5m4qUz0

**Comment by henrysky** [comment]
> I build tf2.0 with GPU support successfully.I use it with keras,strangely,it slower then tf1.14.
> Some code don't work in tf2.0,I couldn't use it.

I have tried but I stuck at error similar to ``calling a __host__ function from a __global__ function is not allowed``, I used ``MSVC++ 14.23 _MSC_VER == 1923 (Visual Studio 2019 Version 16.3)``, cuda 10.1

You mean some python code don't work in tf2.0 or some code in tf2.0 is not working properly?

**Comment by Fannhhyy** [comment]
> > I build tf2.0 with GPU support successfully.I use it with keras,strangely,it slower then tf1.14.
> > Some code don't work in tf2.0,I couldn't use it.
> 
> I have tried but I stuck at error similar to `calling a __host__ function from a __global__ function is not allowed`, I used `MSVC++ 14.23 _MSC_VER == 1923 (Visual Studio 2019 Version 16.3)`, cuda 10.1
> 
> You mean some python code don't work in tf2.0 or some code in tf2.0 is not working properly?

My code don't work,tf2.0 many thing different.
bazel:0.25.0
python:3.6
cuda:10.1
vs:2017
And i use anaconda to configure the python environment,
[tf2.0](https://drive.google.com/file/d/1inRDdmjwJRIU6E_RzNgJe-EDQXpLj0R8/view?usp=sharing)
this is my whl

**Comment by fo40225** [comment]
Tensorflow 2.0.0 uploaded.

Please help to test the function and performance.

https://github.com/fo40225/tensorflow-windows-wheel/tree/master/2.0.0

**Comment by DaTongjie** [comment]
It feels not cool when I use the function 'tf.keras.models.evaluate()', it output a lot of characters like '================================================================================================================================================================================================.....>'.
It don't look same as tf 1.14. Orz!!!

**Comment by Expert73** [comment]
Without CuDNNLSTM, performance dipped 3-5 times. CPU load increased 2 times (70% vs 35%).
How to replace CuDNNLSTM correctly?
LSTM is not a good replacement option.
Please help.

**Comment by henrysky** [comment]
> Without CuDNNLSTM, performance dipped 3-5 times. CPU load increased 2 times (70% vs 35%).
> How to replace CuDNNLSTM correctly?
> LSTM is not a good replacement option.
> Please help.

What is your problem? Can'y you use `tf.compat.v1.keras.layers.CuDNNLSTM`??

**Comment by Expert73** [comment]
> What is your problem? Can't you use `tf.compat.v1.keras.layers.CuDNNLSTM`??

How it's wright in code?
I try:
from tensorflow.compat.v1.keras.layers import CuDNNLSTM
model.add(Embedding(len(my_dict)+1, embedding_dim, input_length=max_len))
model.add(CuDNNLSTM(embedding_dim))

but have:
TypeError: The added layer must be an instance of class Layer. Found: <tensorflow.python.keras.layers.cudnn_recurrent.CuDNNLSTM object at 0x000002722052A408>
